CONTRIBUTING.md

```diff
@@ -63,13 +63,30 @@ We use a centralized approach that leaves `main` as the current development vers
 
 ## Licenses and Copyrights
 
+### Plain English
+
+Legal language can be tricky. Here is what our legal terms are, in plain English. Anywhere that these plain English terms conflict with the legal terms, the legal terms take precedence.
+
+* When you contribute code, you still own the code, but we are free to use it forever.
+* Contributing code to the project does not automatically mean that the end product will be suitable for a particular purpose you may have.
+* If your contribution causes us harm, and you could have prevented it, you are responsible for the harm.
+* We only allow copyleft (GPL) licensed libraries to be used in our copyleft-licensed projects. We don't allow copyleft licenses in non-copyleft projects.
```

Comment:
So no other copy left licenses will be allowed other than GPL and its variants? Is MIT allowed?

I think it should be made clearer what copyleft licenses are. And what is not a copyleft license with examples.

Reply:
GPL was meant as an example, not as an exhaustive list. I'll further clarify.

Reply:
Clarify copyleft please. It's jargon and not plain language.